IQKeyboardManagerSwift/IQKeyboardManager+Position.swift

```diff
@@ -332,7 +333,7 @@ public extension IQKeyboardManager {
             } else {
                 startingScrollIndicatorInsets = unwrappedSuperScrollView.scrollIndicatorInsets
             }
-            #else
+            #elseif swift(>=4.2)
```

Comment:
This elseif was added due to a compilation issue that popped up and this was the lowest runnable version of swift that found the `_startingScrollIndicatorInsets` property